CloudSQLReplicator/google/cloud/pontem/sql/replicator/util/gcp_api_util.py

```diff
@@ -205,12 +321,32 @@ def delete_blob(bucket_name,
   Deletes a Cloud Storage Blob.
 
   Args:
+      bucket_name (str): where blob will be deleted.
+      blob_name (str): blob that will be deleted.
       project (str): Project ID where blob will be deleted.
       credentials (google.auth.Credentials): credentials to authorize client.
-
   """
   storage_client = build_storage_client(project, credentials)
   bucket = storage_client.get_bucket(bucket_name)
   blob = bucket.blob(blob_name)
   blob.delete()
   logging.info('Deleted {}'.format(blob_name))
+
+def grant_read_access_to_bucket(bucket_name,
+                                email,
+                                project=None,
+                                credentials=None
+                           ):
+  """Grants read access to a specific bucket.
+
+    Args:
+        bucket_name (str): name of bucket that the user will be able to read.
+        email (str): email of user who will be granted read privileges.
+        project (str): Project ID where blob will be deleted.
+       credentials (google.auth.Credentials): credentials to authorize client.
```

Comment:
spacing for credentials -- should be moved right one space. shouldn't the linter catch this? is it not being run?